src/users/models.py

```diff
@@ -12,10 +12,12 @@
 # Create your models here.
 class Profile(models.Model):
     YEAR_CHOICES = (
+        ('Incoming', 'Incoming'),
         ('1st', '1st'),
         ('2nd', '2nd'),
         ('3rd', '3rd'),
         ('4th', '4th'),
+        ('5th+', '5th+'),
```

Comment:
sorry I didnt catch this before, but standard style states that you shouldnt have magic strings littered throughout code.  can you make these all CONSTANTS?